01.w_Defects/function_return_value_unchecked.c

```diff
@@ -187,7 +187,13 @@ float (*function_return_value_unchecked_006_func_001())[4]
 	p= (float (*)[]) calloc (1,sizeof(arr));
     if(p!=NULL)
     {
-       memset(p, 1.00, 16);
+    	for (i = 0;i< 4; i++)
```

Comment:
in this case we are performing the usual arithmetic conversions on 1.0 to cast it  to an int, and then setting the first 16 bytes of the buffer to 1, which generates nonsense floats. I don't believe this is what was intended; I think the intended behavior was to set the 16 floats of the array to 1.0.